tests/common/platform/interface_utils.py

```diff
@@ -203,3 +203,35 @@ def get_dpu_npu_ports_from_hwsku(duthost):
             dpu_npu_port_list.append(intf)
     logging.info(f"DPU NPU ports in hwsku.json are {dpu_npu_port_list}")
     return dpu_npu_port_list
+
+
+def get_valid_interfaces(duthost, supported_speeds):
+    """
+    Get interfaces that are up, have an SFP module present, and have supported speeds.
```

Comment:
@vvolam I think interfaces cannot be up if SFP is not present. so its redundant to check for presence

Reply:
Agree! Removed it.